src/main/java/com/google/sps/data/PersistentUserRepository.java

```diff
@@ -59,7 +59,6 @@ public User getUser(com.google.appengine.api.users.User googleUser) {
     User newUser = new User(googleUser.getNickname());
     newUser.setEmail(userEmail);
     newUser.setId(googleUser.getUserId());
-    addUser(newUser);
```

Comment:
Could you do an if-statement like if(user doesn't exist in database) {addUser(newUser)}? I feel like this could be a spot where somehow a user doesn't get added and then we can't figure out why.

Reply:
I'm not doing addUser() bc I'm calling get user just to get the user id and email. Not add.I'm handling the add information on the profileBuilderServlet.

Reply:
Also removed it bc the redirect doesnt properly work with add user